easybuild/tools/options.py

```diff
@@ -813,6 +821,29 @@ def _postprocess_optarch(self):
             else:
                 self.log.info("Keeping optarch raw: %s", self.options.optarch)
 
+    def _postprocess_list_prs(self):
+        """Postprocess --list-prs options"""
+        list_prs_parts = self.options.list_prs.split(':')
+        nparts = len(list_prs_parts)
+
+        if nparts == 0:
+            list_prs_parts = [GITHUB_LIST_PR_STATES[0], GITHUB_LIST_PR_ORDERS[0], GITHUB_LIST_PR_DIRECTIONS[1]]
+        elif nparts == 1:
+            list_prs_parts.extend([GITHUB_LIST_PR_ORDERS[0], GITHUB_LIST_PR_DIRECTIONS[1]])
```

Comment:
another good reason to use dedicated constants for the default values imho, that'll make this more readable :)